source/compose.js

```diff
@@ -15,7 +15,9 @@ export default (config) => {
 
     try {
       Firebase.initializeApp(config)
-    } catch (err) {}
+    } catch (err) {
```

Comment:
This should remain as `} catch (err) {}` for now or else this warning will be thrown when using the library.

There is a plan for not needing this in the future, but for now I would prefer not to throw unnecessary warnings.